### DISCUSSION TITLE
OptiX  and tiny-cuda-nn issue

### DISCUSSION THREAD

**Discussion by yumianhuli2** [NVlabs/instant-ngp#396] [Q&A]
Hello, I get two warnings when compiling with cmake

1、CMake Warning at CMakeLists.txt:177 (message):

OptiX was not found. Neural graphics primitives will still compile and run

correctly. However, SDF training in 'raystab' and 'pathescape' modes will

be significantly slower.

2、CMake Warning at dependencies/tiny-cuda-nn/CMakeLists.txt:112 (message):

Fully fused MLPs do not support GPU architectures of 70 or less. Falling

back to CUTLASS MLPs. Remove GPU architectures 70 and lower to allow

maximum performance

Will these two warnings have a serious impact on the running of the program?

How do I resolve the two warnings separately?

THX！

**Comment by Tom94** [comment]
Hi! To resolve (1), you need to install OptiX as specified in the README and set the corresponding environment variable to the location where you installed it. For (2), I'm afraid your only option is to upgrade to a GPU with newer compute capability.

Both warnings don't affect the _result_ of the program, but mean that it runs slower than it otherwise would.
